advanced_functionality/r_forecast_bring_your_own/R Forecast Bring-Your-Own Container Demo.ipynb

```diff
@@ -0,0 +1,333 @@
+    "\n",
+    "# Create the EndpointConfig\n",
+    "create_endpoint_config_response = sagemaker_client.create_endpoint_config(\n",
+    "    EndpointConfigName = name,\n",
+    "    ProductionVariants=[{\n",
+    "        'InstanceType': INSTANCE_TYPE,\n",
+    "        'InitialInstanceCount': INSTANCE_COUNT,\n",
+    "        'ModelName': name,\n",
+    "        'VariantName': 'AllTraffic'}])\n",
+    "\n",
+    "print(\"EndpointConfigArn: \" + create_endpoint_config_response['EndpointConfigArn'])\n",
+    "\n",
+    "time.sleep(5)\n",
+    "\n",
+    "# Create the Endpoint\n",
+    "create_endpoint_response = sagemaker_client.create_endpoint(\n",
+    "    EndpointName=name,\n",
+    "    EndpointConfigName=name)\n",
+    "\n",
+    "print(\"EndpointArn: \" + create_endpoint_response['EndpointArn'])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Query endpoint status\n",
+    "# The status needs to change to 'InService' before continuing\n",
+    "\n",
+    "describe_endpoint_response = sagemaker_client.describe_endpoint(EndpointName=name)\n",
+    "\n",
+    "print(\"Creating endpoint\", end=\"\")\n",
+    "while describe_endpoint_response['EndpointStatus'] == 'Creating':\n",
+    "    time.sleep(10)\n",
+    "    describe_endpoint_response = sagemaker_client.describe_endpoint(EndpointName=name)\n",
+    "    print(\".\", end=\"\", flush=True)\n",
+    "print(\" done\")\n",
+    "    \n",
+    "assert describe_endpoint_response['EndpointStatus'] == 'InService'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic Forecast Request / Response"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Construct a basic request. The request format is the same as the one used\n",
+    "# used by the DeepAR algorithm,\n",
+    "# see https://docs.aws.amazon.com/sagemaker/latest/dg/deepar-in-formats.html\n",
+    "\n",
+    "toy_time_series = [1, 2, 3, 4, 5]\n",
+    "\n",
+    "request = {\n",
+    "    \"instances\": [\n",
+    "        {\n",
+    "                \"start\": \"2018-01-01\",\n",
+    "                \"target\": toy_time_series\n",
+    "        }\n",
+    "    ]\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# obtain a response from the endpoint\n",
+    "response = sagemaker_runtime_client.invoke_endpoint(\n",
+    "    EndpointName=name,\n",
+    "    ContentType='application/json',\n",
+    "    Body=json.dumps(request)\n",
+    ")\n",
+    "\n",
+    "forecasts = json.loads(response['Body'].read().decode())[\"predictions\"]\n",
+    "forecasts"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot the forecast\n",
```

Comment:
Some of the comments start with caps and some don't. Please make them consistent. Thanks!